docs/rfcs/0000-template.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: <your title>
```

Comment:
I think generally we can remove the angled brackets in the file when used to reference 'your thing' as they're fairly self-explanatory? imo it just adds visual noise without much, if any benefit?